src/ima-state-machine.js

```diff
@@ -540,6 +540,9 @@ function getAdOptions(adEvent: any): Object {
   adOptions.bitrate = ad.getVastMediaBitrate();
   adOptions.bumper = podInfo.getIsBumper() || this._isBumper;
   adOptions.vpaid = this._isVpaid;
+  adOptions.wrapperAdIds = ad.getWrapperAdIds();
+  adOptions.wrapperCreativeIds = ad.getWrapperCreativeIds();
+  adOptions.wrapperAdSystems = ad.getWrapperAdSystems();
```

Comment:
this is breaking the adOptions object that is defined in kaltura-player-js and the `PKAdOptions` type

Reply:
need to use the object type in IMA plugin

Reply:
i updated the type in the playkit and the in the Ad class in kalture-player.js too
https://github.com/kaltura/kaltura-player-js/pull/496
https://github.com/kaltura/playkit-js/pull/608